openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -66,72 +66,41 @@ class RelativeHybridTopologyProtocolResult(gufe.ProtocolResult):
     """Dict-like container for the output of a RelativeHybridTopologyProtocol"""
     def __init__(self, **data):
         super().__init__(**data)
+        # data is mapping of str(repeat_id): list[protocolunitresults]
         # TODO: Detect when we have extensions and stitch these together?
-        if any(len(files['nc_paths']) > 2 for files in self.data['nc_files']):
+        if any(len(pur_list) > 2 for pur_list in self.data.values()):
             raise NotImplementedError("Can't stitch together results yet")
 
-        self._analyzers = []
-        for f in self.data['nc_files']:
-            nc = f['nc_paths'][0]
-            chk = f['checkpoint_paths'][0]
-            reporter = multistate.MultiStateReporter(
-                           storage=nc,
-                           checkpoint_storage=chk)
-            analyzer = multistate.MultiStateSamplerAnalyzer(reporter)
-
-            self._analyzers.append(analyzer)
+        # since we've got no extensions, we can directly take the unit estimates as final
+        self._esimates = [pur_list[0]['estimate'] for pur_list in self.data.values()]
 
     def get_estimate(self):
-        """Free energy difference of this transformation
+        """Average free energy difference of this transformation
 
         Returns
         -------
         dG : unit.Quantity
           The free energy difference between the first and last states. This is
           a Quantity defined with units.
-
-        TODO
-        ----
-        * Check this holds up completely for SAMS.
         """
-        dGs = []
-        # weights = []
-
-        for analyzer in self._analyzers:
-            # this returns:
-            # (matrix of) estimated free energy difference
-            # (matrix of) estimated statistical uncertainty (one S.D.)
-            dG, _ = analyzer.get_free_energy()
-            dG = (dG[0, -1] * analyzer.kT).in_units_of(
-                omm_unit.kilocalories_per_mole)
-
-            # hack to get simulation length in uncorrelated samples
-            # weight = analyzer._get_equilibration_data()[2]
+        # TODO: Check this holds up completely for SAMS.
+        dGs = [pus[0]['estimate'] for pus in self.data.values()]
```

Comment:
I know it's not necessary "right now", but it would be good if we had the opportunity to recalculate in practice - especially when wanting to do certain types of analyses where we might eventually directly pass more data through to pymbar (for example a backwards and forwards truncation analysis for convergence).

Are we ok being clear that this is only **temporary** and we'll move out of this once we spruce up our analysis a bit?